md/release-notes.md

```diff
@@ -3,100 +3,114 @@
+* BS-15827  [UI Designer] preview always display an horizontal scroll bar
+* BS-15823  Installer 7.4.0 displays Copyright 2015 instead of 2016
+* BS-15818  Task form from "Administrator" does not display normally on IE11
+* BS-15797  Variable panel does not resize properly after resizing property panel
+* BS-15780  Text Annotation and Association not supported in Case Visu
+* BS-15770  "Text Annotation" elements are missing when export to BPMN XML
+* BS-15733  fileInput generated with {} by default instead of null 
+* BS-15717  The override popup message provided at UI Designer element import can be inconsistent
+* BS-15716  Cannot access to pending tasks from case details view anymore
+* BS-15712  When a GET Action Button Widget is clicked on a Task Form on the Task List, the event is caught like a Submit Action Button
+* BS-15701  Date widget shows wrong "today" when timezone does not match gmt
+* BS-15696  Error while stopping JBoss bundle
+* BS-15684  The portal starts loading and then reloads
+* BS-15659  Start For should store the name of the person for whom the action is made
+* BS-15579  Infinite $digest Loop when radio buttons is bind to an array
+* BS-15550  Cannot start process in mobile app if skip form is defined
+* BS-15501  REST request retrieve a JSON containing an error and a stack trace
+* BS-15500  Portal error popup contains a stack trace of an exception
+* BS-15484  The manager is set to System when overriding an existing user at organization import
+* BS-15471  Custom widget bond not updated when already in the page
+* BS-15449  Unique index or primary key violation: "UC_CD_SCOPE_NAME_INDEX_9 ON CONTRACT_DATA
+* BS-15431  Security issue - SQL injection [reproducible in JBOSS]
+* BS-15416  Case visualization image does not contain arrows and does not match the process diagram for a process imported from 6.3.6
+* BS-15408  Cache is enabled on REST APIs call
+* BS-15381  Oracle constraint violation on IDX_UQ_PENDING_MAPPING
+* BS-15127  Web Service connector cannot send requests with size > 8 KB on JBOSS only
+* BS-15076  Exception when using BDM queries that return primitives java types
+* BS-15008  ARJUNA016051: thread is already associated with a transaction!
+* BS-14886  Profile API (SP version) createProfileEntry() does not work
+* BS-14816  Executing task lead to violation of unique constraint : idx_uq_pending_mapping
+* BS-14787  Deploying a process having a form undefined and no contract lead to disturbing behavior in the configuration view
+* BS-14327  Cannot install/deploy a connector implementation with a "long" version
+* BS-14276  When a task operation fails the UI form submit receive an exception and fails too
+* BS-13977  A human task can be submitted several times when connector execution is long enough
+* BS-11280  Jasper Connector is not working by default on JBoss
+* BS-10745  Starting a disabled process through a Call Activity should not be allowed
```

Comment:
2 missing bug fixes 
BS-16039 Missing Spanish translations in REST connector 
BS-15970 Studio installer on Windows 32bits use locally installed JRE 6 instead of embedded JRE 7